modules/jdbc-pool/src/main/java/org/apache/tomcat/jdbc/pool/PooledConnection.java

```diff
@@ -335,7 +308,7 @@ protected void connectUsingDriver() throws SQLException {
             }
         }
         if (connection==null) {
-            throw new SQLException("Driver:"+driver+" returned null for URL:"+driverURL);
+            throw new SQLException("Driver:"+parent.getDriver()+" returned null for URL:"+driverURL);
```

Comment:
` throw new SQLException("Driver:" + parent.getDriver() + " returned null for URL:" + driverURL);`